packages/standard_message_codec/example/lib/readme_excerpts.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This file exists solely to host compiled excerpts for README.md, and is not
+// intended for use as an actual example application.
+
+// #docregion Encoding
+import 'dart:typed_data';
+import 'package:standard_message_codec/standard_message_codec.dart';
+// #enddocregion Encoding
```

Comment:
This would excerpt the imports and nothing else. You want these tags around the code.

The reason this didn't fail CI is that you didn't actually follow the wiki instructions to set up excerpting, so the `<?code-excerpt "readme_excerpts.dart (Encoding)"?>` tag is a lie; it's not doing anything. Apparently I need to strengthen the tooling checks to make sure that it's actually set up, but in the meantime could you do a follow-up to fix this where you make sure you do steps 1, 2, 4, and 5 from [this section](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#add-or-converting-a-code-block)?